Framework/PythonInterface/mantid/plots/plotfunctions.py

```diff
@@ -54,6 +54,8 @@ def _setLabels2D(axes, workspace, indices=None):
     axes.set_xlabel(labels[1])
     axes.set_ylabel(labels[2])
     axes.set_title(labels[-1])
+    if workspace.isCommonLogBins():
```

Comment:
This method doesn't exist for MDHistoWorkspace

Reply:
Added a check for that method.